processingchain/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/Workflow.groovy

```diff
@@ -7,6 +7,155 @@ import org.orbisgis.datamanager.JdbcDataSource
 
 @BaseScript ProcessingChain processingChain
 
+
+/**
+ * Load the BDTopo layers from a folder and compute geoindicators (by default the ones needed for the LCZ classification,
+ * for the urban typology classification and for the TEB model). Note that the LCZ classification is performed but
+ * should not be trusted now.
+ *
+ * @param datasource A connection to a database
+ * @param inputFolder The path of the folder that must contains the BDTopo V2 shapefiles
+ * @param distance The integer value to expand the envelope of zone when recovering the data from OSM
+ * (some objects may be badly truncated if they are not within the envelope)
+ * @param indicatorUse List of geoindicator types to compute (default ["LCZ", "URBAN_TYPOLOGY", "TEB"]
+ *                  --> "LCZ" : compute the indicators needed for the LCZ classification (Stewart et Oke, 2012)
+ *                  --> "URBAN TYPOLOGY" : compute the indicators needed for the urban typology classification (Bocher et al., 2017)
+ *                  --> "TEB" : compute the indicators needed for the Town Energy Balance model
+ * @param svfSimplified A boolean indicating whether or not the simplified version of the SVF should be used. This
+ * version is faster since it is based on a simple relationship between ground SVF calculated at RSU scale and
+ * facade density (Bernard et al. 2018).
+ * @param prefixName A prefix used to name the output table (default ""). Could be useful in case the user wants to
+ * investigate the sensibility of the chain to some input parameters
+ * @param mapOfWeights Values that will be used to increase or decrease the weight of an indicator (which are the key
+ * of the map) for the LCZ classification step (default : all values to 1)
+ * @param outputFolder The path of the folder to store the results as geojson files. By default, the files are stored
+ * in the subfolder "results" in the inputFolder
+ *
+ *
+ * @return the path of the folder that contains the geojson files
+ *
+ *
+ * References:
+ * --> Bocher, E., Petit, G., Bernard, J., & Palominos, S. (2018). A geoprocessing framework to compute
+ * urban indicators: The MApUCE tools chain. Urban climate, 24, 153-174.
+ * --> Jérémy Bernard, Erwan Bocher, Gwendall Petit, Sylvain Palominos. Sky View Factor Calculation in
+ * Urban Context: Computational Performance and Accuracy Analysis of Two Open and Free GIS Tools. Climate ,
+ * MDPI, 2018, Urban Overheating - Progress on Mitigation Science and Engineering Applications, 6 (3), pp.60.
+ * --> Stewart, Ian D., and Tim R. Oke. "Local climate zones for urban temperature studies." Bulletin of the American
+ * Meteorological Society 93, no. 12 (2012): 1879-1900.
+ *
+ */
+def BBTOPO_V2() {
+    create({
+        title "Create all geoindicators from OSM data"
+        inputs datasource: JdbcDataSource, inputFolder: String, distance: 0,indicatorUse: ["LCZ", "URBAN_TYPOLOGY", "TEB"],
```

Comment:
We must be clear with the distance parameter (used to merge geometries in the spatial units creation - mostly blocks from buildings).
In the lowest IProcess, distance = 0.01 as default value.
In the geoindicator processing chain, distance = 0.01
At the workflow scale:
- OSM: the distance parameter is not an input anymore
- BdTopo_v2: it is set to 0 by default
Then I propose to set a default value to 0.01 everywhere.

Reply:
Yes right. Call conf is needed

Reply:
Distance is used for envelope